charmcraft/commands/store/__init__.py

```diff
@@ -1051,3 +1051,51 @@ def run(self, parsed_args):
             logger.info("Upload failed with status %r:", result.status)
             for error in result.errors:
                 logger.info("- %s: %s", error.code, error.message)
+
+
+class ListResourceRevisionsCommand(BaseCommand):
+    """List revisions for a resource of a charm."""
+
+    name = 'resource-revisions'
+    help_msg = "List revisions for a resource associated to a charm in Charmhub"
+    overview = textwrap.dedent("""
+        Show size and date for each resource revision in Charmhub.
+
+        For example:
+
+           $ charmcraft resource-revisions
+           Revision    Created at     Size
+           1           2020-11-15   183151
```

Comment:
Is size in bytes or kb?  Would be nice to indicate the unit somewhere.

Reply:
As discussed, will show these values using the `humanize` lib.

Reply:
LGTM, but the doc is out of date now :D